index.bs

```diff
@@ -3036,6 +3036,14 @@ optionally evidence of [=user consent=] to a specific transaction.
         The [=client=] and [=authenticator=] make a best-effort to create a credential of the most preferred type possible.
         If none of the listed types can be created, the {{CredentialsContainer/create()}} operation fails.
 
+        [=[RP]=]'s that wish to support a wide range of [=authenticators=] SHOULD define entries for the following discrete list of signature algorithms:
```

Comment:
I considered adding an "in this order" to this list, but above it mentions "ordered from most preferred to least preferred" so maybe it'd be superfluous?

Reply:
Yeah, I agree that sounds redundant.